wallet-new/src/Cardano/Wallet/WalletLayer.hs

```diff
@@ -485,6 +486,14 @@ data ActiveWalletLayer m = ActiveWalletLayer {
                    -- The payment we need to perform.
                    -> m (Either EstimateFeesError Coin)
 
+    , createUnsignedTx :: InputGrouping
+                       -- An preference on how to group inputs during coin selection
+                       -> ExpenseRegulation
+                       -- Who pays the fee, if the sender or the receivers.
+                       -> Payment
+                       -- The payment we need to perform.
+                       -> m (Either NewTransactionError (Tx, NonEmpty (Address, [Word32])))
```

Comment:
This requires a bit more documentation. What's `Address`, the change address? What are `[Word32]`? A derivation path? Try looking if we already have type synonyms / newtype for a derivation path (perhaps in `cardano-sl-crypto`?), otherwise I guess a type synonym in `V1.Types` might be beneficial!